wordpress-mobile/WordPress-iOS
# Posts & Pages: Add Settings context menu action

Add "Settings" context menu action

To test:

- Select "Settings" from Post (or Pages) context menu
- Verify that when you tap "Cancel", the changes settings are reset to the previous values
- Verify that when you dismiss the settings interactively, the changes settings are reset to the previous values
- Verify that when you tap "Done", the changes are saved after showing a spinner and temporarily disabling interactions

https://github.com/wordpress-mobile/WordPress-iOS/assets/1567433/86252f82-b447-42b6-be33-c138af0ca906

## Regression Notes
1. Potential unintended areas of impact: Posts & Pages
2. What I did to test those areas of impact (or what existing automated tests I relied on): manual
3. What automated tests I added (or what prevented me from doing so): n/a

PR submission checklist:


UI Changes testing checklist:
